packages/css-library/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@department-of-veterans-affairs/css-library",
-  "version": "0.11.1",
+  "version": "0.12.0",
```

Comment:
Can you revert this? It looks like Andrew will be incrementing this when he creates the release [here](https://github.com/department-of-veterans-affairs/component-library/pull/1335).